### PR TITLE
fix: reset head between tests to avoid style bleed

### PR DESCRIPTION
**Pull request by JessicaSachs**
- Closes https://github.com/bahmutov/cypress-vue-unit-test/issues/312 (cannot be transferred)

### User facing changelog
Fixed issue where styles could bleed between tests

### PR Tasks

- [x] Have tests been added/updated?

### CODE REVIEW

File: npm/vue/src/support.js [hunk 1]
```diff
@@ -3,19 +3,16 @@ import unfetch from 'unfetch'
 
 require('@cypress/code-coverage/support')
 
+let headInnerHTML = document.head.innerHTML
+
+// let headInnerHTML
```
Comment (JessicaSachs) [inline npm/vue/src/support.js:8]:
```suggestion
```

File: npm/vue/cypress/component/tailwind/redbox-spec.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ describe('RedBox 1', () => {
   }
 
   it('displays red Hello RedBox', () => {
+    console.log('mounting red box')
```
Comment (JessicaSachs) [inline npm/vue/cypress/component/tailwind/redbox-spec.js:21]:
```suggestion
```

**Comment by JessicaSachs** [comment]
@dmtrKovalenko This is the kinda stuff I wanna merge into a common core

**Comment by dmtrKovalenko** [comment]
What the problem this is solving. Can you make a bug-pinning test that verifies business case?

**Comment by JessicaSachs** [comment]
@dmtrKovalenko will do. In short, it's cleaning up styles and body between test reruns

**Review by elevatebart — APPROVED** [review]

**Pull request merged**